src/components/ContextBar.js

```diff
@@ -39,7 +39,7 @@ const contextBarStyles = {
 
 export const ContextBar = ({
   runtimes, apps, appDataDisks, refreshRuntimes, location, locationType, refreshApps,
-  workspace, persistentDisks, workspace: { workspace: { namespace, name: workspaceName } }
+  workspace, persistentDisks, workspace: { workspace: { namespace, name: workspaceName, googleProject } }
```

Comment:
There is a choice here-- you could pass azureContext and use that to decide if you are on Azure. To me that seems somewhat more straightforward, and clarifies that you have no dependency on the workspace's `googleProject`.

Note that you do get `googleProject` from the currentRuntime, so I'm not clear what will happen with that code once Jupyter actually starts up.

Reply:
it only extracts from `currentRuntime` in the stop function, which is never invoked for azure runtimes. Yeah, I went back and forth on each instance whether azureContext/googleProject is better for making the decision. I'll change it here since generally its easier to read without negation.